src/QMCWaveFunctions/lcao/SoaAtomicBasisSet.h

```diff
@@ -89,8 +90,14 @@ namespace qmcplusplus
       {
         return BasisSetSize;
       }//=NL.size(); 
+      // Set the number of periodic image for the evaluation of the orbitals. 
+      void setPBCImages(std::vector <int> PBCImages)
```

Comment:
I don't like the naming inconsistency between PBCImages and MaxCell. Choose one.
std::vector<int> is not a good type.
TinyVector<int,3> is better.

Reply:
Done